packages/ra-ui-materialui/src/input/ImageInput.stories.tsx

```diff
@@ -83,6 +84,15 @@ export const Required = () => (
     </Wrapper>
 );
 
+export const CustomRemoveIcon = () => (
+    <Wrapper>
+        <ImageInput source="image" RemoveIcon={DeleteIcon}>
+            <FileField source="src" title="title" />
```

Comment:
You should probably use `ImageField` here instead of `FileField`, just like the other stories of this file (and remove the unnecessary import).